### PR TITLE
Added multiple moderators.

### PR DESCRIPTION
**Pull request by roznawsk**

### CODE REVIEW

File: lib/membrane_live/webinars.ex [hunk 1]
```diff
@@ -23,9 +24,21 @@ defmodule MembraneLive.Webinars do
...
+    attrs =
+      attrs
+      |> Map.put("creator_id", creator_id)
+      |> Map.update!("moderators", fn moderators ->
+        if moderators do
+          creator_email = MembraneLive.Accounts.get_email!(creator_id)
+          if creator_email in moderators, do: moderators, else: [creator_email | moderators]
+        else
+          nil
+        end
```
Comment (Rados13) [inline lib/membrane_live/webinars.ex:37]:
```suggestion
    creator_email = MembraneLive.Accounts.get_email!(creator_id)
    attrs =
      attrs
      |> Map.put("creator_id", creator_id)
      |> Map.update!("moderators", fn 
        nil -> nil
        moderators when creator_email in moderators -> moderators
        moderators -> [creator_email | moderators]
      end)
```

File: assets/js/components/dashboard/EventForm.tsx [hunk 2]
```diff
@@ -142,19 +143,43 @@ const PresenterField = ({ inputList, inputSetter }: ListFieldProps) => {
...
 };
 
-const ModeratorField = () => {
+const ModeratorField = ({ inputList, inputSetter }: ListFieldProps) => {
+  const [inputModerator, setInputModerator] = useState<string>("");
+
+  const handleAddButton = () => {
+    inputModerator.trim() && inputSetter([...inputList, inputModerator.trim()]);
+    setInputModerator("");
+  };
```
Comment (Rados13) [inline assets/js/components/dashboard/EventForm.tsx:152]: I think we should verify here if the user passes a proper email. We could use a famous regex for it:
`/^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/`

File: test/membrane_live/webinars_test.exs [hunk 3]
```diff
@@ -55,13 +58,15 @@ defmodule MembraneLive.WebinarsTest do
...
         "presenters" => [],
+        "moderators" => [],
         "start_date" => ~N[2022-07-18 10:20:00],
         "title" => "some updated title"
       }
 
       assert {:ok, %Webinar{} = webinar} = Webinars.update_webinar(webinar, update_attrs)
       assert webinar.description == "some updated description"
       assert webinar.presenters == []
+      assert webinar.moderators == []
```
Comment (Rados13) [inline test/membrane_live/webinars_test.exs:69]: Shouldn't there be always one email, the email of the creator of the event?

File: test/support/fixtures/webinars_fixtures.ex [hunk 4]
```diff
@@ -20,6 +21,9 @@ defmodule MembraneLive.WebinarsFixtures do
     {:ok, webinar} =
       attrs
       |> Enum.into(@default_webinar_attrs)
+      |> Map.update!("moderators", fn moderators ->
+        if user.email in moderators, do: moderators, else: [user.email | moderators]
+      end)
```
Comment (Rados13) [inline test/support/fixtures/webinars_fixtures.ex:26]: Isn't it the same line as in `MembraneLive.Webinars.create_webinar`? Also shouldn't we guarantee that creator's email is always inside the moderator's list?

**Review by Rados13 — CHANGES_REQUESTED** [review]

File: test/membrane_live/webinars_test.exs (hunk 3 above)
Comment (shuntrho) [reply test/membrane_live/webinars_test.exs:69]: You're right, we should never remove the owner from moderators.

File: test/support/fixtures/webinars_fixtures.ex (hunk 4 above)
Comment (shuntrho) [reply test/support/fixtures/webinars_fixtures.ex:26]: This line was excessive - we are adding the creator to moderators in `create_webinar` already.

**Review by pkrucz00 — COMMENTED** [review]
There is a text to fix.

Nonetheless, good work!

File: test/membrane_live_web/controllers/recordings_controller_test.exs [hunk 5]
```diff
@@ -26,7 +26,7 @@ defmodule MembraneLiveWeb.RecordingsControllerTest do
 
     test "lists all recording", %{conn: conn, webinars: webinars} do
       conn = get(conn, Routes.recordings_path(conn, :index))
-      assert json_response(conn, 200)["webinars"] == webinars
+      assert Enum.sort(json_response(conn, 200)["webinars"]) == Enum.sort(webinars)
```
Comment (pkrucz00) [inline test/membrane_live_web/controllers/recordings_controller_test.exs:29]: This still doesn't resolve the issue of failed test check. Sometimes tests don't pass, at least locally.

**Review by pkrucz00 — CHANGES_REQUESTED** [review]

File: test/membrane_live_web/controllers/recordings_controller_test.exs (hunk 5 above)
Comment (shuntrho) [reply test/membrane_live_web/controllers/recordings_controller_test.exs:29]: Right, I had to add sorting in the other test as well.

**Review by pkrucz00 — APPROVED** [review]
Rebase master to resolve conflicts and you're good to go!

**Review by Rados13 — APPROVED** [review]
Looks fine only rebase is needed.

**Comment by shuntrho** [comment]
Work has been suspended, as this feature is not needed for now, and changes have to be made in HLS allowing for multiple moderators.

**Pull request closed without merging**